frontend_tests/node_tests/hotkey.js

```diff
@@ -145,6 +145,7 @@ function stubbing(func_name_to_stub, test_function) {
     global.current_msg_list.empty = return_false;
 
     assert_mapping('@', 'compose.reply_with_mention');
+    assert_mapping('*', 'message_flags.toggle_starred_on_selected_message');
```

Comment:
Also, change assert_mapping to `assert_mapping('*', 'message_flags.toggle_starred');`. Your original function was not necessary.